v3/client.go

```diff
@@ -113,7 +117,6 @@ func NewClient(apiKey, apiSecret string, opts ...ClientOpt) (*Client, error) {
 		apiKey:          apiKey,
 		apiSecret:       apiSecret,
 		serverURL:       "https://api-ch-gva-2.exoscale.com/v2",
```

Comment:
```suggestion
		serverURL:       string(CHGva2),
```
didn't test this

Reply:
I need to modify the generator, to change the default constructor logic, not the generated file. Let me try

Reply:
https://github.com/exoscale/egoscale/pull/610/commits/f2289d326c5869140b956050a4d2f74fe5ea35be